plugin/ochttp/client.go

```diff
@@ -47,6 +47,10 @@ type Transport struct {
 	// for spans started by this transport.
 	StartOptions trace.StartOptions
```

Comment:
Should we deprecate this?

Reply:
If we deprecate this GetStartOptions will be a bad choice of naming. GetX only makes sense when X already exists.